net/socket/tcp_socket_starboard.cc

```diff
@@ -252,7 +252,8 @@ void TCPSocketStarboard::Close() {
 }
 
 void TCPSocketStarboard::StopWatchingAndCleanUp() {
-  bool ok = socket_watcher_.StopWatchingSocket();
+  bool ok = socket_watcher_.UnregisterInterest(kSbSocketWaiterInterestRead |
```

Comment:
There's still a StopWatchingSocket function (afaics) that does the same thing—why not keep using that?